### ISSUE TITLE
Tried to call protected function in combat

### ISSUE THREAD

**Issue by Protuhj** [romdeau23/wow-lfgspamfilter#8]
**Using 1.2.1**

```
20x [ADDON_ACTION_BLOCKED] AddOn 'LFGSpamFilter' tried to call the protected function 'LFGSpamFilterReportHelperButton:Hide()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `Hide'
[string "@LFGSpamFilter\modules/ui.lua"]:40: in function <LFGSpamFilter\modules/ui.lua:37>
[string "=[C]"]: in function `LFGListFrame_SetActivePanel'
[string "@FrameXML\LFGList.lua"]:330: in function `LFGListFrame_FixPanelValid'
[string "@FrameXML\LFGList.lua"]:143: in function <FrameXML\LFGList.lua:138>
```

How to reliably reproduce:

- Open LFG and leave it open
- Go to Dungeons 
- Enter combat
- Close LFG window 

Basically any time you try and close the LFG window while in combat after being on a pane that LFG Spam Filter had modified.

**Comment by romdeau23** [comment]
Should be fixed in 1.2.2, thanks for the report.

**Comment by Protuhj** [comment]
Confirmed it's fixed. Thanks for the quick turnaround!
